examples/demo-vendoring/atmos.yaml

```
@@ -16,6 +16,16 @@ stacks:
     - "**/_defaults.yaml"
   name_pattern: "{stage}"
 
+vendor:  
```

yamllint

[error] trailing spaces

(trailing-spaces)